src/main/java/duke/command/ErrorCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke.command;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.exception.InvalidInputException;
+
+/**
+ * Represents an ErrorCommand and handles methods related to
+ * sending error message to users.
+ */
+public class ErrorCommand extends Command {
```

Comment:
I like how you name your command classes which is pretty clear for me to follow!!

Reply:
Thank you!